src/service/plugins/notification.js

```diff
@@ -397,7 +397,7 @@ var Plugin = GObject.registerClass({
         let file, path, stream, success, transfer;
 
         try {
-            if (!packet.payloadTransferInfo) {
+            if (!packet.payloadSize) {
```

Comment:
Actually, since it's in `_downloadIcon()` specifically, I suppose this could even be
```js
if (packet.payloadSize === 0) { return null; }
```
Since you can't download a 0-byte payload.

Reply:
(Hmm. Would that do the right thing for streams with a `payloadSize` of `-1`, though?)